pkg/login/flow_test.go

```diff
@@ -97,34 +97,49 @@ func TestTriggerMethodOIDCWithLoginServer(t *testing.T) {
 
 func TestTriggerMethodOIDCWithoutLoginServer(t *testing.T) {
 
-	// We're simulating a container environment where the browser can't be opened.
-	// See https://jira.mesosphere.com/browse/DCOS_OSS-5591
-	opener := open.OpenerFunc(func(_ string) error {
-		return errors.New("couldn't open browser, I'm just a container")
-	})
+	fixtures := []struct {
```

Comment:
As a matter of convention, I've generally seen this struct be called `tests`, and then use `tt` or `tc` as the singular in the for range loop. Python prefers fixtures, but in Go it generally is a `tests` struct.